bindings/jupyroot/python/JupyROOT/helpers/utils.py

```diff
@@ -512,9 +545,14 @@ def _getJsCode(self):
             width = self.drawableObject.GetWh()
             options = ""
 
+        if self.isRCanvas:
+            # height = self.drawableObject.GetSize()[0]
```

Comment:
So we don't need to set `height` and `width` here? Why did you leave these lines commented out?

Reply:
I want to change/improve API of `RCanvas` and then enable these lines